### PR TITLE
masking

### PR DESCRIPTION
**Pull request by DNaunin**
Masking the fabric pattern with the design works, there are two options for fabrics and two for design patterns, which are clickable. 
Clearing the canvas does not work yet. 
[See Canvas Page and click on fabrics and designs](https://sewtable-masking-c9k7s1t7wvvlq.herokuapp.com/canvas)

### CODE REVIEW

File: components/canvas/Drawing.module.css [hunk 1]
```diff
@@ -26,8 +25,34 @@
   padding: 0;
 }
 
-.canvas {
+.canvas1 {
```
Comment (m-wiemers) [inline components/canvas/Drawing.module.css:28]: what is the difference between 1 and 2?
I would use a different naming

File: components/canvas/Drawing.module.css [hunk 2]
```diff
@@ -26,8 +25,34 @@
...
+  grid-area: 3 / 1 / 4 / 2;
+  border: none;
+  background: none;
+  width: 75%;
+  height: auto;
+  justify-self: end;
+  padding: 0;
+}
+
+.designbtn2 {
```
Comment (m-wiemers) [inline components/canvas/Drawing.module.css:50]: same here ;-)

File: components/canvas/drawingarea.tsx [hunk 3]
```diff
@@ -1,12 +1,16 @@
 import styles from "./Drawing.module.css";
-import { useRef } from "react";
+import { useEffect, useRef } from "react";
 
 function Drawing() {
   const canvasRef = useRef(null);
+  const canvasRef2 = useRef(null);
 
-  function createPattern(src) {
-    // const src = "/fabric1.svg";
```
Comment (m-wiemers) [inline components/canvas/drawingarea.tsx:8]: I would not put anything out-commented in a PR

**Review by m-wiemers — APPROVED** [review]
THAT's my girl! Good job!

**Review by sweisz — COMMENTED** [review]
GJ 🪡

**Pull request merged**